src/pelicanfs/core.py

```diff
@@ -20,6 +20,7 @@
 from .dir_header_parser import parse_metalink, get_dirlist_loc
 import fsspec.implementations.http as fshttp
 import aiohttp
+import requests
```

Comment:
Instead of bringing in a new HTTP client library, let's use the client in `aiohttp`.